### ISSUE TITLE
Rename key pair files

### ISSUE THREAD

**Issue by christiansmith** [anvilresearch/connect#200]
With future enhancements we'll be managing multiple key pairs for different uses (e.g., signing vs encryption) and supporting multiple algorithms. 

I propose a naming scheme that includes the use, algorithm and part: `sig.rsa.pub.pem` would refer to the public key in a pair used for signing with RSA, for example.

Key rotation is a separate issue, but in order to facilitate this we may want to consider including a timestamp or even a readable date in the name as well: `sig.rsa.pub.1440650111661.pem` or `sig.rsa.pub.2015-08-26.pem`.

A complete "keychain" directory might look like this:

```bash
├── enc.rsa.prv.1440650111661.pem
├── enc.rsa.pub.1440650111661.pem
├── sig.rsa.prv.1440650111661.pem
└── sig.rsa.pub.1440650111661.pem
```

**Comment by christiansmith** [comment]
PR #204 covers this renaming except for the time/date aspect. Adding timestamps complicates loading the files. We should address this in a separate issue as part of a key rotation scheme.

**Comment by vsimonian** [comment]
Merged #204
